part_c1_1:
  name: "part_c1_1"
  mesh_stl_name: "C1.STL"
  pose:
    position:
      x: 0.0
      y: 0.71
      z: 0.9779
    orientation:
      x: 0.70710678118
      y: 0.0
      z: 0.0
      w: 0.70710678118
  relative_pose:  # From robot tool0 to part
    position:
      x: 0.0
      y: 0.22
      z: 0.0
    orientation:
      x: 0.70710678118
      y: 0.0
      z: 0.0
      w: 0.70710678118

part_c1_2:
  name: "part_c1_2"
  mesh_stl_name: "C1.STL"
  pose:
    position:
      x: -0.14
      y: 0.71
      z: 0.9779
    orientation:
      x: 0.70710678118
      y: 0.0
      z: 0.0
      w: 0.70710678118
  relative_pose: # From robot tool0 to part
    position:
      x: 0.0
      y: 0.22
      z: 0.0
    orientation:
#      x: -0.5
#      y: 0.5
#      z: 0.5
#      w: 0.5
      x: -0.7071068
      y: 0.0
      z: 0.0
      w: 0.7071068

part_c1_3:
  name: "part_c1_3"
  mesh_stl_name: "C1.STL"
  pose:
    position:
      x: -0.29
      y: 0.71
      z: 0.9779
    orientation:
      x: 0.70710678118
      y: 0.0
      z: 0.0
      w: 0.70710678118
  relative_pose:  # From robot tool0 to part
    position:
      x: 0.0
      y: 0.22
      z: 0.0
    orientation:
      x: -0.5
      y: 0.5
      z: 0.5
      w: 0.5

part_c2:
  name: "part_c2"
  mesh_stl_name: "C2.STL"
  pose:
    position:
      x: -0.14
      y: 0.87
      z: 0.9779
    orientation:
      x: 0.70710678118
      y: 0.0
      z: 0.0
      w: 0.70710678118
  relative_pose:  # From robot tool0 to part
    position:
      x: 0.0
      y: 0.22
      z: 0.0
    orientation:
      x: -0.5
      y: 0.5
      z: 0.5
      w: 0.5

part_c3:
  name: "part_c3"
  mesh_stl_name: "C3.STL"
  pose:
    position:
      x: -0.22
      y: 1.05
      z: 0.9779
    orientation:
      x: 0.70710678118
      y: 0.0
      z: 0.0
      w: 0.70710678118
  relative_pose:  # From robot tool0 to part TODO: FIX
    position:
      x: 0.0
      y: 0.22
      z: 0.0
    orientation:
      x: -0.5
      y: 0.5
      z: 0.5
      w: 0.5

part_c4:
  name: "part_c4"
  mesh_stl_name: "C4.STL"
  pose:
    position:
      x: 0.02
      y: 1.04
      z: 1.02
    orientation:
      x: 0.70710678118
      y: 0.0
      z: 0.0
      w: 0.70710678118
  relative_pose:  # From robot tool0 to part TODO: FIX
    position:
      x: 0.0
      y: 0.22
      z: 0.0
    orientation:
      x: -0.5
      y: 0.5
      z: 0.5
      w: 0.5

part_c5:
  name: "part_c5"
  mesh_stl_name: "C5.STL"
  pose:
    position:
      x: 0.23
      y: 0.81
      z: 0.9779
    orientation:
      x: 0.5 #0.70710678118
      y: 0.5 #0.0
      z: 0.5 #0.0
      w: 0.5 #0.70710678118
  relative_pose:  # From robot tool0 to part
    position:
      x: 0.0
      y: 0.0
      z: 0.0
    orientation:       # 180 degree rotation about the x-axis
      x: -0.5
      y: 0.5
      z: 0.5
      w: 0.5

part_c6:
  name: "part_c6"
  mesh_stl_name: "C6.STL"
  pose:
    position:
      x: 0.0
      y: 0.86
      z: 0.9779
    orientation:
      x: 0.70710678118
      y: 0.0
      z: 0.0
      w: 0.70710678118
  relative_pose:  # From robot tool0 to part TODO: FIX
    position:
      x: 0.0
      y: 0.0
      z: 0.0
    orientation:       # 180 degree rotation about the x-axis
      x: -0.5
      y: 0.5
      z: 0.5
      w: 0.5

part_c7:
  name: "part_c7"
  mesh_stl_name: "C7.STL"
  pose:
    position:
      x: -0.28
      y: 0.86
      z: 0.9779
    orientation:
      x: 0.5 #0.70710678118
      y: 0.5 #0.0
      z: 0.5 #0.0
      w: 0.5 #0.70710678118
  relative_pose:  # From robot tool0 to part TODO: FIX
    position:
      x: 0.0
      y: 0.0
      z: 0.0
    orientation:       # 180 degree rotation about the x-axis
      x: -0.5
      y: 0.5
      z: 0.5
      w: 0.5

right_panel:
  name: "right_panel"
  mesh_stl_name: "left_right_plate.STL"
  pose:
    position:
      x: -0.06
      y: -0.92
      z: 0.96
    orientation:
      x: 0.7071068
      y: 0.7071068
      z: 0.0
      w: 0.0
  relative_pose:  # From robot tool0 to part
    position:
      x: 0.2
      y: -0.2
      z: -0.05
    orientation:       # 180 degree rotation about the x-axis
      x: -0.7071068
      y: -0.7071068
      z: 0.0
      w: 0.0

back_panel:
  name: "back_panel"
  mesh_stl_name: "front_back_plate.STL"
  pose:
    position:
      x: -0.09
      y: -0.89
      z: 0.98
    orientation:
      x: 0.7071068
      y: 0.7071068
      z: 0.0
      w: 0.0
  relative_pose:  # From robot tool0 to part
    position:
      x: 0.2
      y: -0.2
      z: -0.05
    orientation:       # 180 degree rotation about the x-axis
      x: -0.7071068
      y: -0.7071068
      z: 0.0
      w: 0.0

left_panel:
  name: "left_panel"
  mesh_stl_name: "left_right_plate.STL"
  pose:
    position:
      x: -0.06
      y: -0.92
      z: 1.0
    orientation:
      x: 0.7071068
      y: 0.7071068
      z: 0.0
      w: 0.0
  relative_pose:  # From robot tool0 to part
    position:
      x: 0.2
      y: -0.2
      z: -0.05
    orientation:       # 180 degree rotation about the x-axis
      x: -0.7071068
      y: -0.7071068
      z: 0.0
      w: 0.0

front_panel:
  name: "front_panel"
  mesh_stl_name: "front_back_plate.STL"
  pose:
    position:
      x: -0.09
      y: -0.89
      z: 1.02
    orientation:
      x: 0.7071068
      y: 0.7071068
      z: 0.0
      w: 0.0
  relative_pose:  # From robot tool0 to part
    position:
      x: 0.2
      y: -0.2
      z: -0.05
    orientation:       # 180 degree rotation about the x-axis
      x: -0.7071068
      y: -0.7071068
      z: 0.0
      w: 0.0
